Dockerfile

```diff
@@ -1,6 +1,13 @@
 # Patch and build Olena from Git, then
 # Install OCR-D wrapper for binarization
-FROM ocrd/core
+FROM ocrd/core:v2.62.0 AS base
+ARG VCS_REF
+ARG BUILD_DATE
```

Comment:
that does not help without adding these arguments in the `docker` rule of the makefile (as is done in other repos like ocrd/segment)

Reply:
thanks!